src/integration/common/mediaPlayerEmbed.js

```diff
@@ -0,0 +1,19 @@
+export default () => {
+  describe('Media player embed', () => {
+    describe('A11y', () => {
```

Comment:
I believe the a11y describe belongs with the `I can read the media player title` test